app/services/users.rb

```
@@ -0,0 +1,48 @@
+require 'net/http'
+require 'json'
+# para hacer las pruebas y ejecutar las funciones, cargue el archivo en consola y mande llamar las funciones correspondientes
+def get_data (url)
+  
+  uri = URI(url)
+  # Realiza una petecion HTTP GET al endpoint
+  response = Net::HTTP.get_response(uri)
+  # Revisar si la peticion fue exitosa 
+  if response.is_a?(Net::HTTPSuccess)
+    # Si la respuesta fue correcta, convierte el JSON a un objeto y retorna la informacion recibida para poder procesarla posteriormente.
+    return JSON.load(response.body)
```

RuboCop (1.75.5)

[convention] Prefer `JSON.parse` over `JSON.load`.

(Security/JSONLoad)